### DISCUSSION TITLE
FiveM IMGUI framework

### DISCUSSION THREAD

**Discussion by Zonharo** [supv-cfx/sublime_core-fivem#32] [Ideas]
It is worth looking into, maybe it will be useful in the future. 

**https://github.com/Zonharo/vein**

**Comment by wewesss** [comment]
Hi, we have already use veinUI for sublime_playerCreator (identity) is good, but we doesn't use any external lib, we want make all menu with our own work, only for inventory (because that need a lot of time to make it, so alternativ is to implement bridge for ox_inventory, if we make bridge for ox_inventory we can make bridge for ox_doorlock, target ...) but after that, if we have time we going make inventory like tarkov

**Reply by Zonharo** [reply]
Alright

**Reply by wewesss** [reply]
But thanks for your advice :)
